app/src/main/java/net/gsantner/markor/frontend/MarkorDialogFactory.java

```diff
@@ -80,6 +80,23 @@ public static void showSpecialKeyDialog(Activity activity, GsCallback.a1<String>
         dopt.okButtonText = 0;
         GsSearchOrCustomTextDialog.showMultiChoiceDialogWithSearchFilterUI(activity, dopt);
     }
+    // TODO: this is just a copy of showSpecialKeyDialog, to make minimal invasive changes
+    // but one procedure could be used and an additional paramater, to distinguish between "normal" usage and additional asciidoc usage
+    public static void showAsciidocSpecialKeyDialog(Activity activity, GsCallback.a1<String> callback) {
```

Comment:
should I also remove this function into AsciiDoc specific files?

Reply:
No you can leave that function there. When we see the urge to split it up it's later, affecting all of them.